docs/contributors_guide/Contribution_Procedure.rst

```diff
@@ -55,53 +113,67 @@ Contribution Procedure
      * Use ``git rebase -i`` to squash commits (use ``git rebase -i HEAD~n`` to
        go back ``n`` commits)
 
+   * Make sure to select the option `Allow edits from maintainers`.  This
+     will allow people in the organization to make edits to the pull request.
+
+     * This is the default for any cloned repository
+
 #. `Travis-CI`_ will notice the pull request and run CI tests
 
    * Travis-CI will run tests based on the `.travis.yml`_ file in the
      repository's top-level directory
-   * Travis-CI results can be seen in the PR on GitHub and are posted to the
-     project's `HipChat`_ channel
+   * Travis-CI results can be seen in the pull request on GitHub and are posted
+     to the project's `HipChat`_ channel
+
+#. After passing Travis-CI tests, the GitHub pull request must be reviewed by a
```

Comment:
I haven't seen any documentation on the review process itself.  Perhaps a link to something like https://help.github.com/articles/reviewing-proposed-changes-in-a-pull-request/ would be enough.